.github/workflows/neofs.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish to NeoFS
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  push-to-neofs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set PATH_TO_FILES_DIR
+        run: |
+          echo "PATH_TO_FILES_DIR=output" >> $GITHUB_ENV
+
+      - name: Build Docker image
+        run: make image
+
+      - name: Get pdf file
+        run: make docker/pdf
+
+      - uses: actions/checkout@v4
+      - name: Publish to NeoFS
+        uses: nspcc-dev/gh-push-to-neofs@master
+        with:
+          NEOFS_WALLET: ${{ secrets.NEOFS_WALLET }}
+          NEOFS_WALLET_PASSWORD: ${{ secrets.NEOFS_WALLET_PASSWORD }}
+          NEOFS_NETWORK_DOMAIN: ${{ env.NEOFS_NETWORK_DOMAIN }}
```

Comment:
@vvarg229, why do we have a domain here?

Reply:
> @vvarg229, why do we have a domain here?

How else do you identify which neofs network to use?

Reply:
That's about gateway IIRC.

Reply:
To produce a link to download/view the uploaded thing.